_config.yml

```diff
@@ -37,6 +37,10 @@ flavor: "r"
 # https://github.com/carpentries/workshop-template#creating-extra-pages
 title: "Workshop Title"
 
+## human_language: the language in which the workshop is taught.
```

Comment:
I suggest listing couple of examples after this first line, e.g. "English", "Dutch", etc.